### PR TITLE
Bump sci-biology/ugene version to 1.15.1

### PR DESCRIPTION
**Pull request by stharward**
The source site (unipro.ru) no longer has a downloadable tarball for
1.13.1, the most recent version in this repo.

Cursory testing (compiles, merge, run) on amd64.

### CODE REVIEW

**Comment by marbre** [comment]
Please add an entry to the ChangeLog and I'll merge your request ASAP :)

File: sci-biology/ugene/ugene-1.15.1.ebuild [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	local CONFIG_OPTS
+	if use amd64; then
+		CONFIG_OPTS+=( CONFIG+="x64" )
+	elif use ppc; then
+		CONFIG_OPTS+=( CONFIG+="ppc" )
+	fi
+
+	use cpu_flags_x86_sse2 && CONFIG_OPTS+=( use_sse2 )
+
+	eqmake4 $CONFIG_OPTS || die
```
Comment (jlec) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: no die needed. Functions from eclasses always die internally.

Comment (marbre) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: My fault... I fixed the cpu_flags last weekend and as the 1.15.1 ebuild is identical with 1.13.3, I thought everything should be fine. Unfortunatly, I failed to see the unnecessary die.

@stharward , so please fix this as jlec mentioned.

File: sci-biology/ugene/ugene-1.15.1.ebuild [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+src_configure() {
+	local CONFIG_OPTS
+	if use amd64; then
+		CONFIG_OPTS+=( CONFIG+="x64" )
+	elif use ppc; then
+		CONFIG_OPTS+=( CONFIG+="ppc" )
+	fi
+
+	use cpu_flags_x86_sse2 && CONFIG_OPTS+=( use_sse2 )
```
Comment (junghans) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: This whole thing could be written as:
```
CONFIG_OPTS=(
  usex amd64 x64 ""
  usev ppc
  usex cpu_flags_x86_sse2 use_sse2 ""
)
```

File: sci-biology/ugene/ugene-1.15.1.ebuild [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+DEPEND="
+	dev-qt/qtgui:4"
+RDEPEND="${DEPEND}"
+
+LANGS="cs en ru zh"
+
+src_configure() {
+	local CONFIG_OPTS
+	if use amd64; then
+		CONFIG_OPTS+=( CONFIG+="x64" )
```
Comment (junghans) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: `CONFIG_OPTS+= CONFIG+` huh? Appending to `CONFIG`, which gets appended to `CONFIG_OPTS`, does that make sense?

**Comment by TheChymera** [comment]
I think @stharward might take some time before sorting out ye olde `ChangeLog` and other menial changes, which any of you - and maybe even I - could handle. Why not just pull it and correct what is needed subsequently? 

This is quite a critical pull request since as it stands we have no ugene to offer ATM!

File: sci-biology/ugene/ugene-1.15.1.ebuild (hunk 2 above)
Comment (junghans) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: Fixed a typo above.

File: sci-biology/ugene/ugene-1.15.1.ebuild (hunk 3 above)
Comment (junghans) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: I looked into that and this seems to be a abnormality of `eqmake4` to have `CONFIG+=XXX` as argument.

File: sci-biology/ugene/ugene-1.15.1.ebuild (hunk 2 above)
Comment (junghans) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: I was wrong, the `CONFIG+=` is actually needed, so one could do:
```
eqmake4 \
  $(usex amd64 "CONFIG+=x64" "") \
  $(usex ppc CONFIG+="ppc" "") \
  $(usex cpu_flags_x86_sse2 use_sse2 "")
```

File: sci-biology/ugene/ugene-1.15.1.ebuild [hunk 4]
```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+
+inherit qt4-r2
```
Comment (jlec) [inline sci-biology/ugene/ugene-1.15.1.ebuild]: Could you please switch to qmake-utils.eclass?

**Pull request closed without merging**